src/screening/tb/TbSummary.jsx

```diff
@@ -1,14 +1,21 @@
 import React from "react";
-import {CONCEPTS} from "../../constants";
-import {ObsHistory} from "@openmrs/react-components";
+import PatientLabTests from "../../patient/PatientLabTests";
+import { CONCEPTS } from "../../constants";
+import { ObsHistory } from "@openmrs/react-components";
 
-const TbSummary = props => {
+const TBTestSummary = props => {
   return (
-    <ObsHistory
-      concepts={[CONCEPTS.SymptomPresent,
-        CONCEPTS.SymptomAbsent]}
-    />
+    <div>
```

Comment:
@rhenshaw56 this is for the existing TB Summary page (there will be 2 TB stations) and so does not need to be changed.